atom/common/api/api.mojom

```diff
@@ -64,4 +69,8 @@ interface ElectronBrowser {
 
   [Sync]
   DoGetZoomLevel() => (double result);
+
+  // TODO: move these into a separate interface
+  ShowAutofillPopup(gfx.mojom.RectF bounds, array<mojo_base.mojom.String16> values, array<mojo_base.mojom.String16> labels);
```

Comment:
Is there way to lint .mojom files ? :)

Reply:
probably?

Reply:
looks like not: https://bugs.chromium.org/p/chromium/issues/detail?id=780263